### PR TITLE
[WIP] Refactor the lines over 80 characters

### PR DESCRIPTION
**Pull request by sergiuser1**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -62,40 +68,51 @@ def send_all_prices_message():
     curr_prices = get_all_prices_from_API()
     bot = telegram.Bot(token=BOT_TOKEN)
     try:
-        bot.sendMessage(chat_id=CHAT_ID, text="1€ = " + str(curr_prices[Currency.EURO]) + "mdl, 1$ = " + str(
-                                    curr_prices[Currency.US_DOLLAR]) + "mdl")
+        bot.sendMessage(chat_id=CHAT_ID, text="1€ = {}mdl, 1$ = {]mdl".format(
```
Comment (valeriu33) [inline main.py]: you mistyped second {}

Comment (valeriu33) [reply main.py]: brat, tu cand corectezi?

Comment (sergiuser1) [reply main.py]: Brat, scuz, trebu la univer de prashyt, a s ma uit azi seara.

Comment (sergiuser1) [reply main.py]: Solved

**Comment by valeriu33** [comment]
normal

**Pull request merged**